### ISSUE TITLE
OUTER_TS is 20.0 FOR M5k(GraphSAGE)

### ISSUE THREAD

**Issue by CODE-SUBMIT** [diningphil/gnn-comparison#3]
I RUN the experiments, however, the results is 20.0 for GraphSAGE(RED-M5K)

{"best_config": {"config": {"model": "GraphSAGE", "device": "cuda", "batch_size": 32, "learning_rate": 0.001, "l2": 0.0, "classifier_epochs": 1000, "optimizer": "Adam", "scheduler": null, "loss": "MulticlassClassificationLoss", "gradient_clipping": null, "early_stopper": {"class": "Patience", "args": {"patience": 500, "use_loss": true}}, "shuffle": true, "dim_embedding": 64, "num_layers": 5, "aggregation": "mean", "dataset": "REDDIT-MULTI-5K"}, "TR_score": 19.115831066573453, "VL_score": 20.0}, "OUTER_TR": 19.23931834911077, "OUTER_TS": 20.0}

**Comment by CODE-SUBMIT** [comment]
I also run diffpool for RED-M5k, however it gives the score 20. Is there anything to do to reproduce the results in the paper for RED-M5k.

**Comment by diningphil** [comment]
We added a missing hyper-parameter configuration in the config file. Use Table 7 in the paper as the official reference.
